src/Tribe/Models/Post_Types/Event.php

```diff
@@ -39,6 +40,11 @@ class Event extends Base {
 	 * {@inheritDoc}
 	 */
 	protected function build_properties( $filter ) {
+		// Sanity check.
```

Comment:
This sanity check is a welcome one, but why would this be called while setting up a Venue object? Did you find the origin of that bug?

Reply:
I did not yet. I'm going to ticket it separately.